src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -153,4 +144,36 @@ private function getConfigForPath(array $config, string $path, string $alias)
 
         return $config;
     }
+
+    /**
+     * @return mixed
```

Comment:
this should be array as per `getConfigForPath`

Reply:
I did not trust those methods and their phpdocs enough :) will take another look tomorrow

Reply:
it's a hard type ;) https://github.com/symfony/symfony/blob/cb5e72890b871c896d52abb526cd7ee375f11db1/src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php#L133